packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx

```diff
@@ -9,69 +9,73 @@ import { SelectedItemWithContextMenu } from './Items/SelectedItemWithContextMenu
 import { IRenderFunction } from '../../../Utilities';
 import { IContextualMenuItem } from '../../../ContextualMenu';
 import { IBaseFloatingPickerProps } from '../../../FloatingPicker';
-import { EditingItem } from './Items/EditingItem';
+import { EditingItem, EditingItemFloatingPickerProps } from './Items/EditingItem';
 
 export interface IExtendedPersonaProps extends IPersonaProps {
   key?: React.Key;
-  isValid: boolean;
+  isValid?: boolean;
   blockRecipientRemoval?: boolean;
   shouldBlockSelection?: boolean;
   canExpand?: boolean;
   isEditing?: boolean;
 }
 
-export interface ISelectedPeopleItemProps extends ISelectedItemProps<IExtendedPersonaProps> {
+export interface ISelectedPeopleItemProps<TPersona extends IExtendedPersonaProps = IExtendedPersonaProps>
+  extends ISelectedItemProps<TPersona> {
   onExpandItem?: () => void;
   renderPersonaCoin?: IRenderFunction<IPersonaProps>;
   renderPrimaryText?: IRenderFunction<IPersonaProps>;
 }
 
-export interface ISelectedPeopleProps extends IBaseSelectedItemsListProps<IExtendedPersonaProps> {
-  onExpandGroup?: (item: IExtendedPersonaProps) => void;
+export interface ISelectedPeopleProps<TPersona extends IExtendedPersonaProps = IExtendedPersonaProps>
+  extends IBaseSelectedItemsListProps<TPersona> {
+  onExpandGroup?: (item: TPersona) => void;
   removeMenuItemText?: string;
   copyMenuItemText?: string;
   editMenuItemText?: string;
-  getEditingItemText?: (item: IExtendedPersonaProps) => string;
-  onRenderFloatingPicker?: React.ComponentType<IBaseFloatingPickerProps<IPersonaProps>>;
-  floatingPickerProps?: IBaseFloatingPickerProps<IPersonaProps>;
+  getEditingItemText?: (item: TPersona) => string;
+  onRenderFloatingPicker?: React.ComponentType<EditingItemFloatingPickerProps<TPersona>>;
```

Comment:
Will passing an existing `IBaseFloatingPickerProps` type into here still work without breaking? I think it would since `EditingItemFloatingPickerProps` is a `Pick<IBaseFloatingPickerProps, ... >` but I think it'd be a good sanity check.

Reply:
cc @Adjective-Object

Reply:
yep. For an example: this test case checks in the branch.

https://github.com/OfficeDev/office-ui-fabric-react/blob/3551e64e93e56e7df1488041b7695ab75a97ebc5/packages/office-ui-fabric-react/src/components/ExtendedPicker/examples/ExtendedPeoplePicker.Basic.Example.tsx#L34

https://github.com/OfficeDev/office-ui-fabric-react/blob/3551e64e93e56e7df1488041b7695ab75a97ebc5/packages/office-ui-fabric-react/src/components/ExtendedPicker/examples/ExtendedPeoplePicker.Basic.Example.tsx#L130-L139

an `ISelectedPeopleProp`'s `onRenderFloatingPicker` is bound to FloatingPeople Picker

where FloatingPeoplePicker is defined as an `IBaseFloatingPicker<IPersonaProps ..`, which has a prop types of `IBaseFloatingPickerProps ...`

https://github.com/OfficeDev/office-ui-fabric-react/blob/1b9405b3b64839975e16ee1fad04d7868d9e3b99/packages/office-ui-fabric-react/src/components/FloatingPicker/PeoplePicker/FloatingPeoplePicker.tsx#L11-L13